ThunderCloud/QuizCompletionViewController.swift

```diff
@@ -49,6 +49,14 @@ extension QuizQuestion: Row {
             numberCell.numberLabel.isHidden = true
         }
         
+        numberCell.cellTextLabel?.isHidden = title == nil || title!.isEmpty
```

Comment:
Would `title == nil || title!.isEmpty` be equivalent to `title?.isEmpty ?? true`? That way, if `title == nil` it'll resolve as true, otherwise it'll resolve to the result of `.isEmpty` :)

Reply:
@ryanbourneuk much nicer, thanks!